vignettes/creating_linters.Rmd

```diff
@@ -157,29 +229,37 @@ such as the type, line number, column number, e.g.
 ```r
 expect_lint("blah=1",
   list(message = "assignment", line_number = 1, column_number = 5, type = "style"),
-  assignment_linter())
+  assignment_linter()
+)
 ```
 
 Finally, it is a good idea to test that your linter reports multiple lints if
 needed, e.g.
+
 ```r
 expect_lint("blah=1; blah=2",
   list(
     list(line_number = 1, column_number = 5),
     list(line_number = 1, column_number = 13),
   )
-  assignment_linter())
+  assignment_linter()
+)
 ```
 
 It is always better to write too many tests rather than too few.
 
```

Comment:
Maybe insert a heading "Contributing to {lintr} directly" here?

Reply:
I kept most of the testing section as is, and pulled out the lintr-specific part into its own new section at the end.